### DISCUSSION TITLE
Welcome to distest Discussions!

### DISCUSSION THREAD

**Discussion by Cobular** [JakeCover/distest#39] [General]
## 👋 Welcome!
A new button appeared in the repo's settings this morning, so I guess I'll try this out and see if it's neat or what. Not sure if this is like a chatroom or a forum, we'll see what happens when I post this I guess! 

If you're here for help on the repo, I guess reply and I'll see if this is any better than issues. 

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by johnsturgeon** [comment]
Whoa... yay GitHub.  So, I guess discussions are another kind of issue, 

I'm not a contributor to this project, but I use it for testing my discord bot.  Great testing framework btw.

**Reply by Cobular** [reply]
Cool! Let me know if you think of a neat test to add or anything, winter break is coming up so I might be able to add a thing or two sometime soon.
